pkg/controller/garbagecollector/garbagecollector.go

```diff
@@ -461,14 +466,17 @@ func gcListWatcher(client *dynamic.Client, resource unversioned.GroupVersionReso
 	}
 }
 
-func monitorFor(p *Propagator, clientPool dynamic.ClientPool, resource unversioned.GroupVersionResource) (monitor, error) {
+func (gc *GarbageCollector) monitorFor(resource unversioned.GroupVersionResource) (monitor, error) {
 	// TODO: consider store in one storage.
 	glog.V(6).Infof("create storage for resource %s", resource)
 	var monitor monitor
-	client, err := clientPool.ClientForGroupVersion(resource.GroupVersion())
+	client, err := gc.clientPool.ClientForGroupVersion(resource.GroupVersion())
 	if err != nil {
 		return monitor, err
 	}
+	if rateLimiter := client.GetRateLimiter(); rateLimiter != nil {
+		metrics.RegisterMetricAndTrackRateLimiterUsage(fmt.Sprintf("garbage_collector_monitoring_%s_%s_%s", resource.Group, resource.Version, resource.Resource), rateLimiter)
```

Comment:
RateLimiter metrics are registered here. Note that this function is called for each resource monitored by the garbage collector.